Engine/src/main/java/dk/gruppeseks/bodtrd/engine/Game.java

```diff
@@ -71,34 +73,28 @@ public void create()
         _gamePlugins.addAll(result.allInstances());
         result.allItems();
 
-        for (GamePluginSPI plugin : _gamePlugins)
-        {
+        for (GamePluginSPI plugin : _gamePlugins) {
             plugin.start(_world);
         }
-
+        String path = "C:\\Users\\S\\Documents\\BOD3\\BallOfDutyTheRollingDead\\Common\\assets\\sounds\\JohnCena.mp3";
```

Comment:
take a look at how we reference filepaths in the other components. we dont want to have to change filepaths all the time